grpc/http/forwarders.go

```diff
@@ -179,6 +232,11 @@ func NewStreamForwarder(messageKey func(proto.Message) (string, error)) StreamFo
 				}
 			}
 		}
+
+		if isSSE && os.Getenv("DISABLE_SSE_KEEPALIVE") == "" {
```

Comment:
enable by default yes? 

`os.Getenv("DISABLE_SSE_KEEPALIVE") != "false" `

Reply:
That would have the effect is requiring the environment variable to be set in order to enable the feature - the code as-is supports enable by default when no environment variable is set which I think is desirable.

I can also change this to `os.LookupEnv` and do something like

```go
func isSSEKeepaliveEnabled() {
  value, present := os.LookupEnv("ENABLE_SSE_KEEPALIVE");
  return !present || value == "true"
}
```

I like that a little bit better actually since it supports the case when no environment var has been set, and the logic is a little easier to reason about (I dislike feature flags that disable things versus enabling things personally).

Reply:
if you're confident it'll work, we don't need a flag to turn it off

Reply:
I'm confident it will work in a browser - I was more concerned about unintended side effects for other clients and wanted to provide an escape hatch.  We can also start with no flag and add one in if needed.

Reply:
feature flag removed